vhdbuilder/packer/list-images.sh

```diff
@@ -5,7 +5,7 @@ CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-docker}"
 
 if [[ ${CONTAINER_RUNTIME} == "containerd" ]]; then
     crictl images -o json
-    crictl images -o json | jq -c '[.images | .[] | {id:.id, repoTags:.repoTags, repoDigests:.repoDigests}]' > /opt/azure/containers/image-bom.json
+    crictl images -o json | jq '[.images | .[] | {id:.id, repoTags:.repoTags, repoDigests:.repoDigests}]' > /opt/azure/containers/image-bom.json
```

Comment:
Do we want to store `repoDigests`? It mostly seems to be empty in the sample PR/autonotes run.

Reply:
I swear it's sometimes non-empty, something to debug further I guess...will iterate on this, just trying to get one round of the notes out